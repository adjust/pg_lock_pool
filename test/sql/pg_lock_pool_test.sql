SELECT get_lock_pool(999,3,1);
SELECT get_lock_pool(999,3);
SELECT pg_advisory_unlock(999,1);
BEGIN;
SELECT get_xact_lock_pool(999,3,1);
SELECT get_xact_lock_pool(999,3);
END;