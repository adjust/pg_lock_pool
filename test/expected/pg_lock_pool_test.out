SELECT get_lock_pool(999,3,1);
 get_lock_pool 
---------------
             1
(1 row)

SELECT get_lock_pool(999,3);
 get_lock_pool 
---------------
             1
(1 row)

SELECT pg_advisory_unlock(999,1);
 pg_advisory_unlock 
--------------------
 t
(1 row)

BEGIN;
SELECT get_xact_lock_pool(999,3,1);
 get_xact_lock_pool 
--------------------
                  1
(1 row)

SELECT get_xact_lock_pool(999,3);
 get_xact_lock_pool 
--------------------
                  1
(1 row)

END;
